aat/src/aat/resources/features/F-087/F-087.feature

```diff
@@ -0,0 +1,43 @@
+@F-087
+Feature: Fetch UI definition for Work Basket Inputs By CaseType :: GET /api/display/work-basket-input-definition/{id}
+
+  Background:
+    Given an appropriate test context as detailed in the test data source
+
+  @S-380
+  Scenario: must return the UI definition of work basket inputs for a given case type
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [contains a valid case type id]
+    And it is submitted to call the [Fetch UI definition for Work Basket Inputs By CaseType] operation of [CCD Definition Store]
+    And the response [has the 200 OK code]
```

Comment:
What more and essential is specifically expected in the response of this response? Please mention it.

Reply:
And the response has all the details as expected